tools/ui-components/src/form-group/types.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export interface FormGroupProps extends React.HTMLAttributes<HTMLElement> {
+  componentClass?: typeof React.Component;
+  className?: string;
+  controlId?: string;
+  validationState?: 'success' | 'warning' | 'error' | null;
```

Comment:
I had to add `null` type back, because we are using a `null` state in our codebase.

Here is one of the file that uses `null` validationState 

https://github.com/freeCodeCamp/freeCodeCamp/blob/96bb8e86cdfd3d9d5f9836a7cb2199465b0f3438/client/src/components/settings/internet.tsx